game/scripts/npc/items/item_radiance.txt

```diff
@@ -55,17 +55,17 @@
       "01"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "bonus_damage"                                    "65 80 100 125 160"
+        "bonus_damage"                                    "65 80 120 160 200"
       }
       "03" //has to be wrong in this case else dota will get confused
       {
         "var_type"                                        "FIELD_INTEGER"
-        "aura_damage"                                     "60 90 150 240 360" 
+        "aura_damage"                                     "60 75 150 240 360"
```

Comment:
I know radiance 2 is powerful but that's only because first capture point still exists. Also this number ruins the "aesthetics" of the scaling :(

Reply:
If first cap gets removed it can be reverted.